python/lsst/ts/scheduler/utils/efd_utils.py

```diff
@@ -214,3 +220,82 @@ async def mock_get_fields(topic_name: str) -> List:
     )
 
     return fields[topic_name]
+
+
+async def mock_query(efd_query: str) -> pandas.DataFrame:
+    """Return a data frame for the associated query.
+
+    Parameters
+    ----------
+    efd_query : `str`
+        EFD query.
+
+    Returns
+    -------
+    result : `pandas.DataFrame`
+        Query results.
+    """
+
+    efd_query_match = efd_query_re.match(efd_query)
+
+    if efd_query_match.groups() == (
+        "*",
+        '"efd"."autogen"."lsst.sal.Scheduler.logevent_observation"',
+        "SchedulerID = 1",
```

Comment:
I know there is only one Scheduler and that probably will stay that way. Still, it is scary to hard code an index like that.

Reply:
This is just for testing.. also, we have 2 schedulers, one for each telescope (MT and AT). But again, this is an utility method for testing.